### PR TITLE
Add God Mode

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Adds the God Mode feature.

God Mode allows  sending voice messages without attenuation effects to any player or group of players. The PR functionality includes:
- Integration with Chat Channel. All units matching the Chat Channel criteria will be sent a voice message once the additional PTT is pressed.
- Possibility to define up to 3  group presets that can be activated by additional PTTs.
- Possibility to send text messages to the group presets.
- A set of API  functions to configure the access to God Mode as well as setting  up the group presets.
- TX and RX Notifications with Enable/Disable and Color settings.
- God Volume setting (min. 0.2, cannot mute intentionally).

Additionally introduces `god` speaking type in the C++ land, used for both God Mode and Zeus talk. That resolves an issue with Mumble PTT getting stuck.

### CODE REVIEW

File: addons/sys_godmode/XEH_postInit.sqf [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+if (!hasInterface) exitWith {};
+
+// CBA Event Handlers
+[QGVAR(startSpeaking), {
+    params ["_speakingUnit"];
+
+    // Debug
+    if (GVAR(speakingGods) find _speakingUnit != -1) then {
+        ERROR_1("Tryied to add an already speaking God....",_speakingUnit);
```
Comment (ThymoNL) [inline addons/sys_godmode/XEH_postInit.sqf]: Tried

**Review by ThymoNL — CHANGES_REQUESTED** [review]

**Review by jonpas — APPROVED** [review]

File: addons/sys_godmode/fnc_sendText.sqf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Example:
+ * ["text message", 0] call acre_sys_godmode_fnc_sendTextMessage
+ *
+ * Public: No
+ */
+
+params ["_text", "_group"];
+
+if !([GODMODE_GROUP1] call FUNC(accessAllowed)) exitWith { false };
```
Comment (PabstMirror) [inline addons/sys_godmode/fnc_sendText.sqf:21]: Should this be using `_group` instead?

Comment (jonpas) [reply addons/sys_godmode/fnc_sendText.sqf:21]: It should.

Comment (jonpas) [reply addons/sys_godmode/fnc_sendText.sqf:21]: #1001

**Pull request merged**